e2e-tests/native/wdio-browserstack.conf.ts

```diff
@@ -0,0 +1,74 @@
+import {execSync} from 'child_process'
+import {Capabilities} from '@wdio/types/build/Capabilities'
+
+const getGitBranch = () => {
+    return execSync('git rev-parse --abbrev-ref HEAD').toString().trim()
+}
+
+const getGitHeadReference = () => {
+    return execSync('git rev-parse --short HEAD').toString().trim()
+}
+
+const browserstackCaps = (config: Capabilities): Capabilities => {
+    const isCi = !!process.env.E2E_CI
+    const prefix = isCi ? 'IG CI' : 'IG DEV'
+    const app = config.platformName === 'android' ? process.env.E2E_BROWSERSTACK_APP_ANDROID : process.env.E2E_BROWSERSTACK_APP_IOS
+    return {
+        'bstack:options': {
+            buildName: `${prefix}: ${getGitBranch()}`,
+            sessionName: `${config.browserName?.toLowerCase()}: ${getGitHeadReference()}`,
+            projectName: 'integreat-react-native-app',
```

Comment:
Is this the right project name? I think this was the name from the previous multi repo setup, right?

Reply:
it is the name which is currently used in the e2e tests. Do you think just `integreat-app` makes more sense?

Reply:
No, I was just asking :D Maybe `integreat-app-native` and `integreat-app-web` would make sense and be consistent with our project setup. But I don't mind keeping this reminder to old times :grin: